Formula/tmux.rb

```diff
@@ -32,11 +34,16 @@ class Tmux < Formula
   def install
     system "sh", "autogen.sh" if build.head?
 
+    configure_args = [
+      "--disable-Dependency-tracking",
+      "--prefix=#{prefix}",
+      "--sysconfdir=#{etc}",
+    ]
```

Comment:
> I wasn't offended at all.

Okay, sorry.

> I was more confused as to why I ran the style/convention checker, fixed issues before opening the PR, to then be told I've more style issues that need to be addressed.

Well, again it's safe to ignore me, but I was just trying to help. Style check is only the minimum bar to clear, it's not unreasonable that people more familiar with the code base and its trends can suggest improvements.

In this specific case, you don't need to take my word for it. Just `grep -rI 'args ='` in the repo. You'll realize the majority uses my suggested style. There are some outliers, but if you blame them you'll realize they're generally code that hasn't changed in a long time.

> I was also trying to understand what I had missed, so that I could avoid needing more style fixes after opening a PR in the future.

I don't think having to push style fixes after opening a PR is shameful or anything. Again, I wasn't trying to judge you (and I'm not in the position to judge you), I was just trying to improve the code.